backend/src/api/candidates.js

```diff
@@ -2,6 +2,7 @@ const express = require('express')
 const router = express.Router()
 const { errorWrap } = require('../middleware')
 const { Candidate } = require('../models')
+const { getGithubContributions } = require('../gitScraper')
```

Comment:
wanna move `gitScraper.js`, `util.js`, and `parseSheet.js` into a folder called `utils`